src/main/components/urlBar.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+
+const UrlBar = (props) => {
+  const method = (props.AorB === 'B' ? 'POST' : 'GET')
+  const [selected, setSelected] = useState(method);
+
+  function sendFetch () {
+    const sendingObj = { method: selected, mode: 'cors' };
+    if (props.SourceOrDest === 'dest') sendingObj.body = JSON.stringify(props.requestBody);
+
+    // console.log('hello;', document.querySelector('#urlInput').value);
+
+    fetch(document.querySelector('#urlInput').value, sendingObj)
+      //.then(r => r.json())
+      .then(r1 => console.log(r1));
+  }
+
+  console.log(props)
+
+  return (
+    <div>
+
+      <div > {/* onSubmit={sendFetch} */}
+        {
+          (props.SourceOrDest === 'source') &&
+          <select id='fetchTypeInput' multiple={false} value={selected}
+            onChange={(e) => setSelected(e.target.value)} >
```

Comment:
Not the DRYest way to do this—can we move this to a helper function so it’s reusable between the `source` and `dest` panels? But also… do we really need a select input here at all if the only option is `GET`? May be better to just display that (or a _disabled_ select input)

Reply:
not sure I agree with this one, there is a change i'll make to DRY up a tad but the rest I'll leave for now!